docs/reusables/credential-request-code-snippet.mdx

```diff
@@ -5,5 +5,6 @@
   required?: boolean, // @deprecated - Use mandatory field instead. If the credential is required
   mandatory?: 'yes' | 'no' | 'if_available' // if the credential is required, not required, or required if available.
   description?: string // a description field that is shown to the user; can be used to explain why the information is being requested
+  children?: CredentialRequest[] // this field contains the list of sub-credentials relevant to the parent credential that are being requested (eg: FullNameCredential(FirstNameCredential, MiddleNameCredential LastNameCredential), AddressCredential(Line1Credential, Line2Credential, CityCredential, StateCredential, ZipCodeCredential, CountryCredential))
```

Comment:
This example "syntax" is confusing. What are you trying to show here?

Reply:
Oh, I think just make sense if we look when is render 

<img width="991" alt="image" src="https://github.com/UnumID/UnumID.github.io/assets/11665444/7f27d816-9188-4320-920d-5b44c6828486">